source/dotnet/Library/AdaptiveCards.Rendering.Wpf.Xceed/XceedTimeInput.cs

```diff
@@ -41,6 +41,12 @@ public static FrameworkElement Render(AdaptiveTimeInput input, AdaptiveRenderCon
                 timePicker.Style = context.GetStyle("Adaptive.Input.Time");
                 timePicker.DataContext = input;
 
+                if (input.IsRequired && input.ErrorMessage == null)
+                {
+                    context.Warnings.Add(new AdaptiveWarning((int)AdaptiveWarning.WarningStatusCode.NoWarningForValidatedInput,
+                        "Inputs with validation should include an ErrorMessage"));
```

Comment:
Is there a common place in the shared .NET library that we could house this string as a variable to reference in all these places vs needing to hardcode the literal each place?